Riot/Modules/Onboarding/AuthenticationCoordinatorState.swift

```diff
@@ -0,0 +1,41 @@
+// 
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import MatrixSDK
+
+@available(iOS 14.0, *)
+struct AuthenticationCoordinatorState {
+    // MARK: User choices
+    // var serverType: ServerType = .unknown
```

Comment:
Yes, this is correct. I'll investigate `SignMode` when I implement Login and serverType may or may not be a relevant property to us on iOS, I think it will probably go, but would prefer to keep it here as a reminder to look into.